iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSerDe.java

```diff
@@ -257,6 +260,13 @@ public SerDeStats getSerDeStats() {
     return null;
   }
 
+  @Override
+  public void handleJobLevelConfigurations(HiveConf conf) {
```

Comment:
nit: I think there's no plural form of the word configuration or at least we shouldn't use it here IMHO.
Similarly for jobConfs -> jobConf or serdeDefaultConf perhaps serdeDefaultSettings?